build.gradle

```diff
@@ -15,10 +15,20 @@ ext {
     akkaVersion = '2.5.19'
 }
 
+def readCurrentVersion = {
+  def out = new ByteArrayOutputStream()
+
+  exec{
+      commandLine "./version"
```

Comment:
I was just wondering why we are not using Scala but I guess the `version` script is from Marathon.

Reply:
It's very useful to have the script, IMO. Experience has informed me that moving all of our build automation to Scala is undesirable